dev-util/jenkins-bin/files/jenkins-bin-systemd.confd

```diff
@@ -0,0 +1,68 @@
+# Config file for /etc/init.d/jenkins
+
+#
+# Directory where Jenkins store its configuration and working
+# files (checkouts, build reports, artifacts, ...).
+#
+JENKINS_HOME="/var/lib/jenkins/home"
+
+#
+# Options to pass to java when running Jenkins.
+#
+JENKINS_JAVA_OPTIONS="-Djava.awt.headless=true"
+
+#
+# Port Jenkins is listening on.
+#
+JENKINS_PORT="8080"
+
+# Secure port Jenkins is listening on
+JENKINS_SSL_PORT="8443"
+
+#
+# Debug level for logs -- the higher the value, the more verbose.
+# 5 is INFO.
+#
+JENKINS_DEBUG_LEVEL="5"
+
+#
+# Whether to enable access logging or not.
+#
+JENKINS_ENABLE_ACCESS_LOG="no"
+
+#
+# Maximum number of HTTP worker threads.
+#
+JENKINS_HANDLER_MAX="100"
+
+#
+# Maximum number of idle HTTP worker threads.
+#
+JENKINS_HANDLER_IDLE="20"
+
+#
+# Pass arbitrary arguments to Jenkins.
+# Eg.: --httpListenAddress=127.0.0.1 --prefix=/jenkins
+# Full option list: java -jar jenkins.war --help
+#
+JENKINS_ARGS=""
+
+#
+# User
+#
+JENKINS_USER=jenkins
+
+
+
+#
+# Pid File 
+#
+JENKINS_PIDFILE="/var/run/jenkins.pid"
+
+#
+# War File
+#
+JENKINS_WAR="/opt/jenkins/jenkins.war"
+
+# Unified command to escape all dashes
+JAVA_COMMAND_LINE="/usr/bin/java ${JENKINS_JAVA_OPTIONS} -DJENKINS_HOME=${JENKINS_HOME} -jar ${JENKINS_WAR} --daemon --logfile=/var/log/jenkins/jenkins.log --httpPort=${JENKINS_PORT} --httpsPort=${JENKINS_SSL_PORT} --debug=${JENKINS_DEBUG_LEVEL} --handlerCountMax=100 --handlerCountMaxIdle=20 --accessLoggerClassName=winstone.accesslog.SimpleAccessLogger --simpleAccessLogger.format=combined --simpleAccessLogger.file=/var/log/jenkins/access_log ${JENKINS_ARGS} "
```

Comment:
This should be JENKINS_COMMAND_LINE to match up with the usage in the systemd file (thanks to mceier on IRC).

Reply:
Which makes me wonder, have you tested this?